bin/webpack-configs.js

```diff
@@ -95,10 +97,6 @@ const getCoreConfig = ( options = {} ) => {
 			path: path.resolve( __dirname, '../build/' ),
 			library: [ 'wc', '[name]' ],
 			libraryTarget: 'this',
-			// This fixes an issue with multiple webpack projects using chunking
-			// overwriting each other's chunk loader function.
-			// See https://webpack.js.org/configuration/output/#outputjsonpfunction
-			jsonpFunction: 'webpackWcBlocksJsonp',
```

Comment:
What has changed so that the JSONP functions from different blocks no longer conflict and don't need to be assigned different names?

Reply:
`jsonpFunction` is no longer an option in webpack 5, it's now replaced with `chunkLoadingGlobal`.
However, I couldn't find any reference for this `webpackWcBlocksJsonp`. I'm not sure what we're trying to solve here.
The [original PR](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/903/files) that introduced this (2 years ago) provides very little rationale about what was happening here.
Deleting it doesn't seem to break anything in my testing.